lambdas/src/Server.ts

```diff
@@ -120,6 +121,8 @@ export class Server {
 
     // Functionality for Explore use case
     this.app.use('/explore', initializeExploreRoutes(express.Router(), env.getBean(Bean.DAO), contentClient))
+
+    this.app.use('/third-party-integrations', initializeThirdPartyIntegrationsRoutes(theGraphClient, express.Router()))
```

Comment:
Should we update the docs?

Reply:
Can we have a `@beta` channel in the docs? I'm afraid that people consider unstable APIs as part of the protocol and then those should be working forever after that